### PR TITLE
Add DB and common model generators

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR kicks off our new approach to defining / using generators for tests!

### CODE REVIEW

File: src/test/java/marquez/common/models/CommonModelGenerator.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  private CommonModelGenerator() {}
+
+  private static final int BOUND = DatasourceType.values().length;
+  private static final Random RANDOM = new Random();
+
+  public static OwnerName newOwnerName() {
+    return newOwnerNameWith("test_owner" + newId());
+  }
+
+  public static OwnerName newOwnerNameWith(String value) {
```
Comment (ashulmanWeWork) [inline src/test/java/marquez/common/models/CommonModelGenerator.java]: Should we add validation, such as `@NotBlank` or `@NonNull`? Should we also make it final?

File: src/test/java/marquez/db/models/DbModelGenerator.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
+package marquez.db.models;
```
Comment (ashulmanWeWork) [inline src/test/java/marquez/db/models/DbModelGenerator.java]: Please add the license info.

File: src/test/java/marquez/common/models/CommonModelGenerator.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  public static Description newDescription() {
+    return newDescriptionWith("test_desciption" + newId());
+  }
+
+  public static Description newDescriptionWith(String value) {
+    return Description.fromString(value);
+  }
+
+  private static int newId() {
```
Comment (sshah-wework) [inline src/test/java/marquez/common/models/CommonModelGenerator.java]: we will have something similar in all generators, so let's find a way to standardize.

File: src/test/java/marquez/db/models/DbModelGenerator.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+    if (wasUpdated) {
+      builder.updatedAt(newTimestamp());
+      builder.currentVersion(UUID.randomUUID());
+    }
+
+    return builder.build();
+  }
+
+  public static Instant newTimestamp() {
```
Comment (sshah-wework) [inline src/test/java/marquez/db/models/DbModelGenerator.java]: potentially something to make common as well across Generators

File: src/test/java/marquez/common/models/CommonModelGenerator.java (hunk 1 above)
Comment (wslulciuc) [reply src/test/java/marquez/common/models/CommonModelGenerator.java]: For test code we can relax input validation. Let's say an invalid `value` is passed to `newOwnerNameWith`, an exception will be thrown by `wnerName.fromString()`. So, we're covered :)

File: src/test/java/marquez/common/models/CommonModelGenerator.java (hunk 3 above)
Comment (wslulciuc) [reply src/test/java/marquez/common/models/CommonModelGenerator.java]: I think we might be okay just keeping `newId()` in this class as it's used to provide unique names. As we expand the usage of generators, we can begin moving common helper methods into a base generator.

**Pull request merged**